src/network/networkd-manager.c

```diff
@@ -690,6 +694,11 @@ int manager_start(Manager *m) {
 
         assert(m);
 
+        sysctl_add_monitor(m);
+        r = drop_capability(CAP_SYS_ADMIN);
+        if (r < 0)
+                log_warning_errno(r, "Failed to drop CAP_SYS_ADMIN: %m");
+
```

Comment:
hmm, so you also added CAP_BPF to the unit file, and do not drop it here. Is there a reason for that?

I presume so, hence please extend the comment about this too.

also, I think it would be nice if we'd drop CAP_BPF in case `m->sysctl_link` is NULL here, im case the BPF stuff didn't work. (which might quite likely happen, for example if this is a networkd that is run in some container or so, where BPF is not available for some reason)

Reply:
I thought that further operations, like map update and unload still requires CAP_BPF.
But I'm not sure about this.

Reply:
please figure this out, and then either drop that cap here too, or add a comment explaining why we don't.

Reply:
Ok, I tried and I can use bpf these functions after dropping CAP_BPF.
I'm dropping both here unconditionally.